src/script/search/SearchRepository.ts

```diff
@@ -169,16 +169,16 @@ export class SearchRepository {
     isHandle?: boolean,
     maxResults = SearchRepository.CONFIG.MAX_SEARCH_RESULTS,
   ): Promise<User[]> {
+    const domain = query.includes('@') ? query.substr(query.lastIndexOf('@') + 1) : undefined;
+    const name = domain ? query.substr(0, query.indexOf('@')) : query;
```

Comment:
```suggestion
   const [name, domain] = query.split('@');
```
i am pretty sure this would do the same in a more readable manner. WDYT?

Reply:
It should indeed, let me run a few tests